Sources/PointFree/Stripe.swift

```diff
@@ -319,10 +325,14 @@ private func stripeDataTask<A>(_ path: String, _ method: Method = .get)
   -> EitherIO<Prelude.Unit, A>
   where A: Decodable {
 
-    return pure(stripeUrlRequest(path, method))
+    let task: EitherIO<Prelude.Unit, A> = pure(stripeUrlRequest(path, method))
       .flatMap { jsonDataTask(with: auth <| $0, decoder: stripeJsonDecoder) }
       .map(tap(AppEnvironment.current.logger.debug))
       .withExcept(tap(AppEnvironment.current.logger.error) >>> const(unit))
+
+    return method.isPost
+      ? task.retry(maxRetries: 10)
+      : task
```

Comment:
I could also do `.retry(maxRetries: method.isPost ? 10 : 0)` but I also like this

Reply:
Do we wanna do the same for `DELETE`s? Maybe the logic should be:

``` swift
return method.isGet
  ? task
  : task.retry(maxRetries: 10)
 ```

Reply:
`DELETE`s should be idempotent to begin with, so those should be ok. stripe's docs specifically call out POST's as the main culprit. 

however, perhaps better is to just switch here so that we have to consider this again when adding a new verb.